android-agent/src/main/java/io/opentelemetry/android/internal/services/PreferencesService.java

```diff
@@ -15,7 +15,7 @@
  * <p>This class is internal and not for public use. Its APIs are unstable and can change at any
  * time.
  */
-public class PreferencesService implements Service {
+public class PreferencesService {
```

Comment:
Could consider renaming to just `Preferences`.

Reply:
Thanks, I've just made the changes.